crates/biome_cli/tests/snapshots/main_cases_config_path/set_config_path_to_directory.snap

```diff
@@ -0,0 +1,61 @@
+---
+source: crates/biome_cli/tests/snap_test.rs
+expression: content
+---
+## `config/biome.jsonc`
+
+```jsonc
+
+        {
```

Comment:
I think this isn't probably formatted because of the trailing commas. We should review the snapshot internals and apply the correct options

Reply:
This is because the command I use in the test case will only format files in the `src` directory. Do you want me to change it?

https://github.com/biomejs/biome/blob/de596fdd8b52b9ac60455b9a380685ae1b12b340/crates/biome_cli/tests/cases/config_path.rs#L43

Reply:
No need. The snapshot test applies a specific formatting to the configuration file:

https://github.com/biomejs/biome/blob/57eda2822f59b0a9fecdc3dcec6e98baa310af72/crates/biome_cli/tests/snap_test.rs#L62-L71

Reply:
Ahhh I get you now. I can make a small fix for this :)

Reply:
After some digging, I think the main reason they're not formatted is that the snapshot builder failed to detect the config files. This is expected because to test `--config-file` I've to put them to a place (sub dir) where Biome cannot auto discover.

https://github.com/biomejs/biome/blob/57eda2822f59b0a9fecdc3dcec6e98baa310af72/crates/biome_cli/tests/snap_test.rs#L345-L354

To address this problem we'll have to modify the snapshot payload. It seems not a small fix. So I just manually formatted the content.

I also updated the json parser options, it doesn't fix these two particular cases, but it does work on some other test cases.

Reply:
No worries, we can do it later